RELEASE.md

```diff
@@ -1,26 +1,69 @@
-# _Product Model_ - Security Finance trade types
+# *CDM Model - Equity Products*
 
 _Background_
 
-The Asset Refactoring initiative (see [#2805](https://github.com/finos/common-domain-model/issue/2805)) is seeking to improve the Product Model to address some long-standing issues and to ensure the continued extensibility to additional financial products and markets. A proposal has been agreed - through a cross-industry Task Force - to implement this remodelling in the CDM.
-
-This release includes two minor adjustments following three planned major tranches of work in CDM 6 to implement the refactored model.
+This release contains modifications required to accomodate Equity and Exotic Products under individual Asset Classes
 
 _What is being released?_
 
-An optional attribute exists on the `AssetPayout` data type to uniquely identify the different types of securities financing trade types, such as a repurchase transaction or a buy/sell-back.  The name of this attribute has been updated to have a broader potential scope and not limit just to repos.  Specifically, the attribute `repoType` has been renamed to `tradeType` and its data type has been renamed from `RepoTypeEnum` to `AssetPayoutTradeTypeEnum`.  The values of the enumerator have not been changed.
+This release creates following modifications:
+- a new qualification function `Qualify_Equity_OtherOption` to add qualification for Exotic Options
```

Comment:
For instance here, explain that the new qualification is making use of the `nonStandardisedTerms` attribute. The attribute is also being negatively tested in the pre-existing qualification functions to prevent redundant qualification.